lib/sinatra/base.rb

```diff
@@ -1936,7 +1925,7 @@ class #{self.class}
             </style>
           </head>
           <body>
-            <h2>Sinatra doesn&rsquo;t know this ditty.</h2>
+            <h2>Sinatra doesn’t know this ditty.</h2>
```

Comment:
@kgrz Oh I'm way more concerned with HTML than typography. I'll defer to the typography concern though.